app/assets/javascripts/application.js

```diff
@@ -1,2 +1,9 @@
+//= require jquery
 // = require govuk/all.js
+//= require govuk_publishing_components/components/cookie-banner
+//= require govuk_publishing_components/lib/cookie-functions
+//= require analytics
+//= require cookies
+window.CookieSettings.start()
+window.GOVUK.analyticsInit()
```

Comment:
Since these are not run asynchronously I would suggest initialising the UI components first and then the cookie settings and analytics.